python/cython_build_ext.py

```diff
@@ -59,46 +64,74 @@ class cython_build_ext(_build_ext, object):
-    profile : bool, default=False
-        Write hooks for Python profilers into the compiled C code.
+    cython_exclude : list of str
+        When passing glob patterns as module_list, you can exclude certain
+        module names explicitly by passing them into the exclude option.
     embedsignature : bool, default=False
         If set to True, Cython will embed a textual copy of the call signature
         in the docstring of all Python visible functions and classes. Tools
         like IPython and epydoc can thus display the signature, which cannot
         otherwise be retrieved after compilation.
-    cython_exclude : list of str
-        When passing glob patterns as module_list, you can exclude certain
-        module names explicitly by passing them into the exclude option.
     gdb_debug : bool, default=False
         Passes the `gdb_debug` argument to `cythonize()`. Setting up debugging
         for Cython can be difficult. See the debugging docs here
         https://cython.readthedocs.io/en/latest/src/userguide/debugging.html
+    language_level : {"2", "3", "3str"}, default="2"
+        Globally set the Python language level to be used for module
+        compilation. Default is compatibility with Python 2. To enable Python 3
+        source code semantics, set this to 3 (or 3str)
+    linetrace : bool, default=False
+        Write line tracing hooks for Python profilers or coverage reporting
+        into the compiled C code. This also enables profiling. Default is
+        False. Note that the generated module will not actually use line
+        tracing, unless you additionally pass the C macro definition
+        ``CYTHON_TRACE=1`` to the C compiler (e.g. using the setuptools option
+        define_macros). Define ``CYTHON_TRACE_NOGIL=1`` to also include nogil
+        functions and sections.
+    profile : bool, default=False
+        Write hooks for Python profilers into the compiled C code.
     """
     user_options = [
-        ('language-level=', None,
-         'Sets the python language syntax to use "2", "3", "3str".'),
-        ("binding", None,
+        ("annotate=",
+         None,
+         "Passes the `annotate` argument to `cythonize()`. See the Cython "
+         "docs for more info."),
+        ("binding",
+         None,
          "Sets the binding Cython compiler directive. See the Cython docs for "
          "more info."),
-        ("profile", None,
-         "Sets the profile Cython compiler directive. See the Cython docs for "
-         "more info."),
-        ("embedsignature", None,
-         "Sets the `embedsignature` Cython compiler directive. See the Cython "
-         "docs for more info."),
-        ("cython-exclude=", None,
+        ("cython-exclude=",
+         None,
          "Sets the exclude argument for `cythonize()`. See the Cython docs for"
          " more info."),
-        ("gdb-debug=", None,
+        ("embedsignature",
```

Comment:
Do we need embedsignature?

Reply:
Similar to `profile` this currently exists in 0.16. You are only seeing the diffs here because I reorganized all of the properties to be in alphabetical order.